test/verify/check-shell-host-switching

```diff
@@ -62,10 +62,16 @@ class HostSwitcherHelpers:
           return true;
         })""", address)
 
-    def add_machine(self, b, address, known_host=False, pixel_label=None):
+    def add_machine(self, b, address, known_host=False, pixel_label=None, user=False):
+        # Trim ssh:// from the address for known_host message
+        if address[:6] == "ssh://":
+            address = address[6:]
```

Comment:
That's cheating :grin:  The new test actually want to type the `ssh://` prefix into the `#add-machine-address` field. You only want to strip it for checking the "You are connecting to .." message.

Reply:
@martinpitt also, should the test log in directly, or 'Authorize SSH key'?

Reply:
The SSH key gets generated without a password in `setup_ssh_auth()`, i.e. it calls `ssh-keygen` with `-N ''`. So I suppose something is legitimately broken with these `ssh://` prefixes? Does it work without the prefix? (That's exactly why we want to test all cases :grin: )

Feel free to push your current code if you want me to have a closer look -- you probably have local modifications over what's pushed here, so I can only guess what's wrong.